src/data/Profile.json

```diff
@@ -58,6 +58,34 @@
       "LinkedIn": "https://linkedin.com/in/alicesmith"
     }
   },
+  {
+		"name": "André Lopes",
+		"location": "Lisbon, Portugal",
+		"bio": "I've been coding for as long as I remember and have been working in the area since 2018! I'm currently working as a Fullstack Developer at a fortune500 in Lisbon and trying to learn more everyday about the latest techs.",
+		"avatar": "https://github.com/AndreJorgeLopes.png",
+		"portfolio": "https://github.com/AndreJorgeLopes",
+		"skills": [
+			"Next.js",
+			"Node.js",
+			"React.js",
+			"Vue.js",
+			"TypeScript",
+			"Storybook",
+			"Gatsby",
+			"Drupal",
+			"GraphQL",
+			"MongoDB",
+			"Postgres",
+			"Redis",
+			"RabbitMQ",
+			"AWS",
+			"GCP"
+		],
```

Comment:
@AndreJorgeLopes I appreciate your PR but you have so many skills, and some skills are rendering out of the profile. Can you only add less than 8 skills.